packages/zotero.rb

```diff
@@ -0,0 +1,127 @@
+# Thanks to @agkphysics of AUR repository for his zotero pkgbuild, which heavily based this package.
+
+require 'package'
+
+class Zotero < Package                 # The first character of the class name must be upper case
+  description 'A free, easy-to-use tool to help you collect, organize, cite, and share your research sources..'
+  homepage 'https://www.zotero.org/'
+  version '6.0.26'
+  license 'AGPL3' # license of source
+  compatibility 'x86_64'
+  source_url 'https://aur.archlinux.org/cgit/aur.git/plain/zotero.desktop?h=zotero'
+  source_sha256 'eab76db7a56a4d9aaa17baaf240b82fcf57944a4ddf8ef1b58cc64182426cedc'   # Use the command "sha256sum"
+
+  depends_on 'dbus_glib'            # packages required by this package
+  depends_on 'gtk3'
+  depends_on 'desktop_file_utils'
+  depends_on 'nss'
+  depends_on 'libxt'
+  depends_on 'perl'
+  depends_on 'node' => :build      # packages with "=> :build" are only required if you're building from source 
+  depends_on 'git' => :build
+  depends_on 'zip' => :build
+  depends_on 'unzip' => :build
+  depends_on 'python3' => :build
+  depends_on 'curl' => :build
+  depends_on 'wget' => :build
+  depends_on 'rsync' => :build
+  depends_on 'nodebrew' => :build
+
```

Comment:
Something like this should help ensure that nodebrew is installed and configured properly:
```
  def self.prebuild
    # Nodejs is needed for this install, and nodejs is installed by
    # nodebrew.
    @nodebrew_version = `nodebrew ls`.lines.first.chomp
    if @nodebrew_version.include?('not installed')
      puts 'nodejs is not configured!'.lightred
      puts "Please run 'crew postinstall nodebrew' for more info.".yellow
      abort
    else
      puts "nodebrew version is #{@nodebrew_version}".yellow
    end
  end
```